packages/cli/src/lib/snapshot/snapshot.spec.ts

```diff
@@ -98,6 +101,31 @@ describe('Snapshot', () => {
     );
   };
 
+  const doMockSnapshotReporter = (
+    snapshotReportStatus: SnapshotReportStatus
+  ) => {
+    let handlerToCall: (
+      this: SnapshotReporter
+    ) => void | Promise<void> = () => {};
+    mockedSnapshotReporter.prototype.setReportHandler.mockImplementation(
```

Comment:
It's better to mock method individually, otherwise, strangely, `jest.mock.instances[0]` will not be `jest.mock.returns[0].value`, which mean that if you try to check if X is called with an instance of a mock, well, you're outta luck